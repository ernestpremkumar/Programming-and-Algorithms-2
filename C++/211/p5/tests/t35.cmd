1 0 tests/t35.in results/t35.myout
