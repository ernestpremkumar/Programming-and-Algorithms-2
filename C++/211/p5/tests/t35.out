registers[0] = $250
time = 1
